node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always, { null: 'ignore' } ],
+    'no-else-return': [ warn, { allowElseIf: false } ],
+    'yoda': error,
+    'no-throw-literal': error,
+    'no-self-compare': error,
+    'no-return-assign': error,
+    'no-loop-func': error,
+    'valid-jsdoc': warn,
+    'for-direction': error,
+    'no-buffer-constructor': error,
+    'no-new-require': error,
+    'no-path-concat': error,
+    'comma-spacing': error,
```

Comment:
Enforce a space after a comma
https://eslint.org/docs/rules/comma-spacing
```js
// Old
const arr = [1 , 2];
const func = (a,b ,c) => a + b + c
// New
const arr = [ 1, 2 ]
const func = (a, b, c) => a + b + c
```

Reply:
What about when de-structuring (probably agree)?
```js
// old
const [ a,, c ] = [ 1, 2, 3 ]
// new
const [ a, , c ] = [ 1, 2, 3 ]
```

Reply:
Yeah IMO the extra space when de-structuring makes it clearer that the extra comma is there intentionally and it's not a programmer mistake

Reply:
I don't like 

```
[ 1, 2, 3 ]
```

I usually write

```
[1, 2, 3]
```

but for hashes I prefer the spaces

```
{ a: "b", c: "d" }
```

but probably it's just me

Reply:
@makevoid you're the only :-1: on this one at the moment - what was your concern? This rule isn't about bracket spacing, it enforces a single space after a comma `[ 1,2,3 ]` => `[ 1, 2, 3 ]` (bracket spacing is already enforced)

Reply:
ok then it's a separate issue